source/channels/organize-conversations.rst

```diff
@@ -83,16 +67,23 @@ Select **Threads** at the top of the channel sidebar to see all your followed th
 
 Select **Unreads** to filter your followed threads by only those with unread replies.
 
-.. image:: ../images/crt-thread-view.png
+.. image:: ../images/crt-thread-view.jpg
   :alt: Select Threads in the channel sidebar to see all thread updates in your Threads View.
 
 Tutorial video
 ---------------
 
-For additional details about working with the beta release of Collapsed Reply Threads, see our `tutorial video on YouTube <https://www.youtube.com/watch?v=hnD0Zj-mIbs>`_. 
+For additional details about working with Collapsed Reply Threads, see our `tutorial video on YouTube <https://www.youtube.com/watch?v=hnD0Zj-mIbs>`_. 
 
 .. raw:: html
 
    <div style="position: relative; padding-bottom: 50%; height: 0; overflow: hidden; max-width: 100%; height: auto;">
       <iframe src="https://www.youtube.com/embed/hnD0Zj-mIbs" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 95%;"></iframe>
    </div>
+
+Enable Collapsed Reply Threads
+------------------------------
+
+See our `configuration settings <https://docs.mattermost.com/configure/configuration-settings.html#collapsed-reply-threads>`__ documentation for details on configuring the default availability of collapsed reply threads. From Mattermost v7.0, Collapsed Reply Threads are enabled by default for all new Mattermost deployments.```
+
+Depending on how your System Admin has configured **Collapsed Reply Threads** for your workspace, it may already be enabled for you, or you may be able to `enable this feature for your account <https://docs.mattermost.com/channels/channels-settings.html#collapsed-reply-threads>`__ by going to **Settings > Display > Collapsed Reply Threads**. 
```

Comment:
Not sure if this line needed the formatting that I deleted in my previous comment but noting it here in case it did!